### PR TITLE
add acceleration to particle info [com1DFA]

### PR DESCRIPTION
**Pull request by awirb**
address issue #617

### CODE REVIEW

File: avaframe/out3Plot/outCom1DFA.py [hunk 1]
```diff
@@ -102,6 +102,40 @@ def plotTrackParticle(outDirData, particlesList, trackedPartProp, cfg, dem):
         # ani.save("testTrackAlr1.gif", writer=writer)
 
 
+def plotTrackParticleAcceleration(outDirData,trackedPartProp, cfg, Tsave):
+    """ Plot time series of tracked partcles
```
Comment (fso42) [inline avaframe/out3Plot/outCom1DFA.py:106]:
```suggestion
    """ Plot time series of tracked particles
```

File: avaframe/out3Plot/outCom1DFA.py [hunk 2]
```diff
@@ -102,6 +102,40 @@ def plotTrackParticle(outDirData, particlesList, trackedPartProp, cfg, dem):
...
+    Parameters
+    ----------
+    outDirData: str
+        path to output directory
+    particlesList: list
+        list or particles dictionaries
```
Comment (fso42) [inline avaframe/out3Plot/outCom1DFA.py:112]: this is not in the input list, remove?

File: avaframe/out3Plot/outCom1DFA.py [hunk 3]
```diff
@@ -102,6 +102,40 @@ def plotTrackParticle(outDirData, particlesList, trackedPartProp, cfg, dem):
...
+    trackedPartProp: dict
+        dictionary with time series of the wanted properties for tracked
+        particles
+    cfg : dict
+        configuration read from ini file
+    dem: dict
+        dem dictionary with normal information
```
Comment (fso42) [inline avaframe/out3Plot/outCom1DFA.py:119]: exchange for Tsave

File: docs/com1DFAAlgorithm.rst [hunk 4]
```diff
@@ -131,8 +131,33 @@ There are then different ways to place the particles in the cells. This is decid
...
+
+    - ``ux``, ``uy``, ``uz`` - velocity components [ms-1]
+
+    - ``uAcc`` - approximation for particle acceleration between each computational time step (simply computed by (velocityMagnitude_t0 - velocityMagnitude_t1) / dt)
+
+    - ``travelLengthXY`` - traveled length of particle accumulated over time in xy plane
+
+    - ``travelLengthXYCor`` - traveledLengthXY corrected with the angle difference between the slope and the normal
+
+    - ``travelLengthXYZ`` - traveled length of a particle accumulated over time in xyz
```
Comment (fso42) [inline docs/com1DFAAlgorithm.rst:152]: Pending our discussion: rename to trajectoryLengthXX

**Review by fso42 — CHANGES_REQUESTED** [review]

File: avaframe/out3Plot/outCom1DFA.py (hunk 3 above)
Comment (awirb) [reply avaframe/out3Plot/outCom1DFA.py:119]: actually the trackedParticle dict has its own time info

**Review by fso42 — APPROVED** [review]

**Comment by fso42** [comment]
Standardtests ok

**Pull request merged**